src/components/paradox/results/personality/career/modal/jobs.js

```diff
@@ -1,13 +1,40 @@
 import PropTypes from "prop-types";
+import {
+  faBuilding,
+  faLocationDot,
+  faSuitcase
+} from "@fortawesome/free-solid-svg-icons";
+import Icon from "lib/helpers/icon";
 import style from "./style.scss";
 
 function Jobs({jobs}) {
   return (
     <div className={style.list}>
       {jobs.map((job) => (
         <div className={style.listItem} key={job.id}>
-          <div className={style.title}>{job.title}</div>
-          <div className={style.description}>{job.description}</div>
+          <div className={style.job}>
+            <div className={style.jobDetails}>
+              <Icon className={style.jobTitleIcon} icon={faSuitcase} />
+              <div>
+                <div className={style.title}>
+                  {job.title}
+                </div>
+                <div className={style.description}>
```

Comment:
I'm not completely sure, but I think most of the job fields could come back blank since we use a few different services. So for description and below, you probably want to check if it's present before displaying the block

Reply:
This is a good catch! Description blocks now only render if the corresponding field is present!